index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css ">
+        <script src="https://kit.fontawesome.com/5116129a4d.js"
+            crossorigin="anonymous"></script>
+        <title>AALA</title>
+    </head>
+
+    <body>
+
+        <section class="sec1">
+            <div class="desktopdarkheader">
+                <div class="dark-items">
+                    <ul>
+                        <li><i class="fa-brands fa-facebook-f"></i></li>
+                        <li><img src="images/twitter.png" alt=""></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="desktoplighterheader">
+                <div class="desktopheader">AALA 2022</div>
+
+                <nav class="references">
+                    <a href="about.html" class="about">About</a>
+                    <a href="#mainprogramme" class="programme">Program</a>
+                    <a href="">Join</a>
+                    <a href="#sec4">Sponsor</a>
+                    <a href="">News</a>
+                </nav>
+            </div>
+
+
+
+            <div class="container">
+                <div id="container">
+                    <button class="btn"><i class="fa-solid fa-bars navbtn"></i></button>
+                    <div class="txt">
+                        <h1 class="text">Hello! Welcome</h1>
+                    </div>
+                    <div id="description">
+                        <h1 class="description">Annual Art And Literary
+                            Appreciation 2022!</h1>
+                    </div>
+                    <div id="texts">
+                        <p class="texts">An appreciation of literature, musical
+                            giants and their excellence, arts,
+                            craft, sculpture, traditions, history that predates
+                            time herself, and so much more.</p>
+                    </div>
+                    <div id="date">
+                        <h2 class="date">2022.9.12(MON) ~ 13(TUE) </h2>
+                    </div>
+                    <h3 class="location">@ NewYork city hall</h3>
+                </div>
+            </div>
```

Comment:
Kindly reduce the size of text and spacing between elements in this section so that it fits the screen and matches the given design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/190650178-06f8c7f8-ab0b-472a-90ff-1de2a20c9b4e.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/190650270-8adf0fff-2c3c-4736-b2cc-ff1a90c9c7e3.png)

Reply:
done thanks